spec/rester/rspec_spec.rb

```diff
@@ -4,20 +4,63 @@
   describe '/v1/tests' do
     context 'GET' do
       context 'With some context' do
-        it { is_expected.to eq stub_response }
-        it { is_expected.to eq [200, '{"string":"some_string","integer":1,"float":1.23,"symbol":"some_symbol","bool":true,"method":"search"}'] }
+        it { is_expected.to include stub_response }
+        it {
+          is_expected.to include(
+            "string" => "some_string",
+            "integer" => 1,
+            "float" => 1.23,
+            "symbol" => "some_symbol",
+            "bool" => true,
+            "method" => "search"
+          )
+        }
+      end
```

Comment:
I see. So in `rspec.rb` line 56, it should be `JSON.parse(raw_service_response.last, symbolize_keys: true)`?